modules/apps/ccv/child/keeper/relay.go

```diff
@@ -30,8 +32,14 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data c
 		k.SetParentChannel(ctx, packet.DestinationChannel)
 	}
 	// Set PendingChanges to be flushed and the unbonding time and unbonding packet.
-	// TODO: Get PendingChanges and update the pending changes if they already exist
-	k.SetPendingChanges(ctx, data)
+	currentChanges, exists := k.GetPendingChanges(ctx)
+	if !exists {
+		currentChanges = &ccv.ValidatorSetChangePacketData{ValidatorUpdates: []abci.ValidatorUpdate{}}
+	}
+	k.SetPendingChanges(ctx, ccv.ValidatorSetChangePacketData{
+		ValidatorUpdates: utils.AccumulateChanges(currentChanges.ValidatorUpdates, newChanges.ValidatorUpdates),
+	})
```

Comment:
```suggestion
        var pendingChanges []abci.ValidatorUpdates{}
	currentChanges, exists := k.GetPendingChanges(ctx)
	if !exists {
	    pendingChange = newChanges.ValidatorUpdates
	} else {
	    pendingChanges = utils.AccumulateChanges(currentChanges.ValidatorUpdates, newChanges.ValidatorUpdates
	}
	k.SetPendingChanges(ctx, ccv.ValidatorSetChangePacketData{ValidatorUpdates: pendingChanges}),
	})
```

This makes a lot more sense to me, but up to you